CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.15.1 (July 2020)
+- Android 10 support
```

Comment:
Any interesting improvement for the users included in Android 10?

Reply:
we can include that we support TLS 1.3 from now on. Something like:

`Android 10: Support to TLS 1.3`